pkg/i18n/english.go

```diff
@@ -1094,7 +1103,7 @@ func EnglishTranslationSet() TranslationSet {
 		Fixup:                                "Fixup",
 		SureFixupThisCommit:                  "Are you sure you want to 'fixup' the selected commit(s) into the commit below?",
 		SureSquashThisCommit:                 "Are you sure you want to squash the selected commit(s) into the commit below?",
-		Squash:                               "Squash",
```

Comment:
Sorry: we should keep the original Squash item here and add a new SquashMerge item. Squashing and squash merging are two separate things and we do not want to conflate them

Reply:
ok done